lib/chitter.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class Chitter
+
+
+
+  def self.all
```

Comment:
could you create a setup_test_database spec in order to avoid writing out the if statements in each method. This would clear it up nicely. E.g. with the bookmarks 

def reset_table
  connection = PG.connect(dbname: "bookmark_manager_test")

  # Clear the bookmarks table
  connection.exec("TRUNCATE bookmarks;")
end

def add_bookmarks
  connection = PG.connect(dbname: "bookmark_manager_test")
  # Add the test data
  connection.exec("INSERT INTO bookmarks VALUES(1, 'http://www.makersacademy.com','Makers Academy');")
  connection.exec("INSERT INTO bookmarks VALUES(2, 'http://www.destroyallsoftware.com','Destroy all software');")
  connection.exec("INSERT INTO bookmarks VALUES(3, 'http://www.google.com','Google');")
end